pkg/cnservice/cnclient/client.go

```diff
@@ -65,19 +86,21 @@ type ClientConfig struct {
 func NewCNClient(cfg *ClientConfig) error {
 	var err error
 	cfg.Fill()
-	Client = &CNClient{config: cfg}
-	Client.requestPool = &sync.Pool{New: func() any { return &pipeline.Message{} }}
+	client = &CNClient{config: cfg}
+	client.requestPool = &sync.Pool{New: func() any { return &pipeline.Message{} }}
```

Comment:
where to release message to the pool

Reply:
I miss the acquire logic and release logic. and I add it now.
it will call the acquireMessage when we try to send pipeline to remote-node, and release it when we over the remote-run at `cnserviceDriver.go`.

you can see my next comment. I point out the code.